src/core.js

```diff
@@ -49,7 +49,13 @@ function createModule( name, testEnvironment ) {
 		testsRun: 0,
 		unskippedTestsRun: 0,
 		childModules: [],
-		suiteReport: new SuiteReport( name, parentSuite )
+		suiteReport: new SuiteReport( name, parentSuite ),
+
+		// Pass along `skip` property from parent module, in case there is one,
+		// to childs. And use own otherwise.
+		// This property will be used to mark own tests and tests of child suites
+		// as `skipped`.
+		skip: parentModule !== null && parentModule.skip || skip
```

Comment:
Right now, what happens is as follow:
* The tests of a normal module that was defined inside a `skipped/todo` will be treated as either `todo` to `skipped` based on the modifier of the parent module.
* If a module was defined as `only`, it will be the only module that will run. But, in case it is inside a `todo` or `skipped` module, its tests will be marked as either `todo` or `skipped` based on the modifier of the parent module.
* If a module was defined as `skipped` and is inside a `todo` module, its tests will be left intact.
* If a module was defined as `todo` and was inside a `skipped` module, its tests will be treated as `skipped` not as `todo`s.

The last two points were based on the order I first defined in my first PR:
```js
if ( this.module.skip ) {
  // ...
} else if ( this.module.todo ) {
  // ....
}
```